.github/workflows/validate-examples-code.yaml

```diff
@@ -0,0 +1,25 @@
+name: "[TEST-IGNORE] Verify Swagger Examples"
```

Comment:
While I agree the new name is more descriptive, I lean towards keeping the existing names to reduce churn, since many spec authors (and ARM reviewers) are familar with the existing names.

Reply:
Understood I'll change this.